Symfony/CS/Tests/FixerTest.php

```diff
@@ -205,6 +205,7 @@ public function getFixersPriorityCases()
             array($fixers['php_unit_strict'], $fixers['php_unit_construct']),
             array($fixers['unary_operators_spaces'], $fixers['logical_not_operators_with_spaces']),
             array($fixers['unary_operators_spaces'], $fixers['logical_not_operators_with_successor_space']),
+            array($fixers['short_bool_cast'], $fixers['spaces_cast']),
```

Comment:
Symfony integration test tests whole level.
The priority relation should be tested by integration test as well, it is not related with level